src/iceberg_extension.cpp

```diff
@@ -147,25 +150,18 @@ static unique_ptr<Catalog> IcebergCatalogAttach(StorageExtensionInfo *storage_in
 		return std::move(catalog);
 	}
 
-	// Check no endpoint type has been passed.
-	if (!endpoint_type.empty()) {
-		throw IOException("Unrecognized endpoint point: %s. Expected either S3_TABLES or GLUE", endpoint_type);
```

Comment:
nit: I would like to keep this in for UX. `ENDPOINT_TYPE` is for glue and s3tables, but may later be expanded.

Reply:
There is no reason to change this in this PR

Reply:
This check is unnecessary for the non-polaris code path as ENDPOINT TYPE is not used there.

Reply:
Ok it is back